flink-runtime/src/main/java/org/apache/flink/runtime/io/network/api/writer/RecordWriter.java

```diff
@@ -20,7 +20,7 @@
 
 import org.apache.flink.core.io.IOReadableWritable;
 import org.apache.flink.runtime.accumulators.AccumulatorRegistry;
-import org.apache.flink.runtime.event.AbstractEvent;
+import  org.apache.flink.runtime.event.AbstractEvent;
```

Comment:
Good catch. I'm wondering how that happened though as I don't import stuff manually... :dart: